modules/virt-about-hostpath-provisioner.adoc

```diff
@@ -6,22 +6,8 @@
 [id="virt-about-hostpath-provisioner_{context}"]
 = About the hostpath provisioner
 
-The hostpath provisioner is a local storage provisioner designed for
-{VirtProductName}. If you want to configure local storage for
-virtual machines, you must enable the hostpath provisioner first.
+When you install the {VirtProductName} Operator, the hostpath provisioner (HPP) Operator is automatically installed. The HPP is a local storage provisioner designed for {VirtProductName}. If you want to configure local storage for virtual machines, you must enable the HPP first.
 
-When you install the {VirtProductName} Operator, the hostpath provisioner Operator
-is automatically installed. To use it, you must:
+In {VirtProductName} 4.10, the HPP is implemented using a Kubernetes CSI driver. The CSI driver enables you to configure local storage for your virtual machines with much less performance overhead, such as a restart of all {product-title} nodes because the `MachineConfig` object is no longer used to write to the directory on the host.
 
-* Configure SELinux:
-** If you use {op-system-first} 8 workers, you must create a `MachineConfig`
-object on each node.
-** Otherwise, apply the SELinux label `container_file_t` to the persistent volume (PV) backing
-directory on each node.
-* Create a `HostPathProvisioner` custom resource.
-* Create a `StorageClass` object for the hostpath provisioner.
```

Comment:
Changed to:

```
When you install the {VirtProductName} Operator, the hostpath provisioner (HPP) Operator is automatically installed. The HPP is a local storage provisioner designed for {VirtProductName}. If you want to use the HPP, you must:

* Create the HPP custom resource.
* Create a StorageClass object for the HPP.
```

Reply:
Maybe let's keep 'HostPathProvisioner' because it is what is expected to be in the yaml's 'kind', so we'll not confuse the users with acronyms at this point :)

Reply:
So, we can specify the full name when referring to the operator, but otherwise we writers are encouraged to use acronyms especially for common components/features. As long as the acronym is well defined at the start of the content, it helps to minimize the word count, facilitates translation, and encourages users to read the doc. Whether or not to use acronyms varies on what the acronym is and how often the word it represents is used. but either way we have to defer to our style guidelines.